src/spec/payid-uri.md

```diff
@@ -50,242 +63,202 @@ informative:
-   a 'payid' URI on its own, e.g., as the value of the 'href' attribute
-   of an HTML anchor element.  For example, there is no behavior
-   specified in this document for a 'payid' URI used as follows:
-
-   ```
-    <a href='payid:bob$example.com'>find out more</a>
-   ```
-
-   Any protocol that uses 'payid' URIs is responsible for specifying how
-   a 'payid' URI is employed in the context of that protocol (in
-   particular, how it is dereferenced or resolved; see [RFC3986]).  As a
-   concrete example, an "Account Information" application of the
-   WebFinger protocol [RFC7033] might take a 'payid' URI, resolve the
-   host portion to find a WebFinger server, and then pass the 'payid' URI
-   as a parameter in a WebFinger HTTP request for metadata (i.e., web
-   links [RFC5988]) about the resource. For example:
-
-   ```
-    GET /.well-known/webfinger?resource=payid%3Abob%24example.com HTTP/1.1
-   ```
-
-   In the above example, the service retrieves the metadata associated with
-   the payment account identified by that URI and then provides that
-   metadata to the requesting entity in an HTTP response.
-
-   If an application needs to compare two 'payid' URIs (e.g., for
-   purposes of authentication and authorization), it MUST do so using
-   case normalization and percent-encoding normalization as specified in
-   Sections 6.2.2.1 and 6.2.2.2 of [RFC3986]. In addition, the 'acctpart' 
-   is case-insensitive and therefore should be normalized to lowercase. 
-   For example, the URI `PAYID:aLICE$www.EXAMPLE.com` is equivalent to
-   `payid:alice$www.example.com`.
+   In addition, a 'payid' is case-insensitive and therefore should be
+   normalized to lowercase.  For example, the URI 
+   "PAYID:aLICE$www.EXAMPLE.com" is equivalent to 
+   "payid:alice$www.example.com".
+   
+   Note that both the 'acctpart' and 'host' components of a 'payid' may
+   contain one or more dollar-sign characters. However, because a 'host' 
+   SHOULD also be a valid DNS domain, that portion of a 'payid' will
+   generally not include a dollar-sign. Therefore, applications should
+   always search for the last dollar-sign when attempting to parse a 'payid' 
+   URI into its two component parts.
+
+# Examples                                                              
+   As an example, a user with an account name of "apollo" at a wallet
+   service "wallet.example.com" can be identified by a URI using the 'payid' 
+   scheme via the following construction: 
+   
+     'payid:apollo$wallet.example.com'
+
+   One possible PayID scenario is for an account to be registered with a 
+   payment service provider using an identifier that is associated with some 
+   other service provider. For example, a user with the email address 
+   "alice@example.net" might register with a wallet website whose domain
+   name is "wallet.example.com". In order to facilitate payments to/from
+   Alice, the wallet service provider might offer alice a PayID using alice's 
```

Comment:
alice and Alice are are used interchangeably. It should be `alice` for the payid and `Alice` to refer to the person Alice?

Reply:
Fixed.